app/models/employee.rb

```diff
@@ -8,4 +8,8 @@ class Employee < ApplicationRecord
 
   has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', dependent: :destroy, inverse_of: :giver_of_kudo
   has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :receiver_of_kudo
+
+  def kudos_count
+    received_kudos.all.count
```

Comment:
1) you don't need `.all`. 
2) IMO the name might be a bit confusing. This isn't kudos_count (kudos IMO means all kudos), but received_kudos_count or even `points`.